src/main/resources/stubs/time/sleep.gobra

```diff
@@ -0,0 +1,13 @@
+// Copyright 2009 The Go Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in https://golang.org/LICENSE
+
+// Signatures for the public declarations in file
+// https://github.com/golang/go/blob/master/src/time/sleep.go
+
+package time
+
+// Sleep pauses the current goroutine for at least the duration d.
+// A negative or zero duration causes Sleep to return immediately.
+decreases _ if d < 0
```

Comment:
```suggestion
decreases _ if d <= 0
```
Furthermore, we could think about termination even for positive arguments because even a 10sec sleep eventually terminates...

Reply:
Unfortunately, the official documentation does not say anything about `Sleep` terminating eventually. In fact, the documentation says exactly what is said in the comment.

I agree that a non-terminating sleep would be mostly useless but I think we should not provide stronger specs than what is provided in the official docs.

Reply:
Sorry, you're right. I did not watch out for `at least` and relied too much on my intuition